tests/test_devices.py

```diff
@@ -69,15 +70,15 @@ def test_get_device():
     project_id = "prj_123"
     responses.add(
         responses.GET,
-        api_url(f"/v1/devices/{id}"),
+        api_url(f"/v1/devices/{id}?projectId={project_id}"),
```

Comment:
do we have a test which still ensures `get_device` functions properly without a `project_id`?